packages/sdk/python/src/relayfile/setup.py

```
@@ -0,0 +1,686 @@
+    normalized = _normalize_connection_id(connection_id)
+    if normalized is None:
+        raise MissingConnectionIdError("connectionId is required", "missing_connection_id")
+    return normalized
+
+
+def _read_json(response: httpx.Response) -> Any:
+    content_type = response.headers.get("content-type", "")
+    if "application/json" in content_type:
+        try:
+            return response.json()
+        except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)